packages/ckeditor5-image/src/image/converters.js

```diff
@@ -64,12 +64,12 @@ export function viewFigureToModel() {
 /**
  * Converter used to convert the `srcset` model image attribute to the `srcset`, `sizes` and `width` attributes in the view.
  *
+ * @param {'image'|'imageInline'} imageType The type of created image.
```

Comment:
```suggestion
 * @param {'image'|'imageInline'} imageType The type of the image.
```

Reply:
I'm not sure if it should be "the" or "an".

Reply:
done